deepface/modules/preprocessing.py

```diff
@@ -33,6 +35,10 @@ def load_image(img: Union[str, np.ndarray]) -> Tuple[np.ndarray, str]:
     # The image is a base64 string
     if img.startswith("data:image/"):
         return load_base64(img), "base64 encoded string"
+    
+    # The video is a base64 string
+    if img.startswith("data:video/"):
+        return loadVideo_base64(img), "base64 encoded string"   
```

Comment:
return type of load image is image itself as numpy array. load video base64 returns that numpy array as many numpy arrays.

Won't this fail here: https://github.com/serengil/deepface/blob/master/deepface/modules/detection.py#L77C5-L77C8